cluster-autoscaler/estimator/binpacking_estimator.go

```diff
@@ -51,27 +57,74 @@ func NewBinpackingNodeEstimator(predicateChecker *simulator.PredicateChecker) *B
 // still be maintained.
 // It is assumed that all pods from the given list can fit to nodeTemplate.
 // Returns the number of nodes needed to accommodate all pods from the list.
-func (estimator *BinpackingNodeEstimator) Estimate(pods []*apiv1.Pod, nodeTemplate *schedulernodeinfo.NodeInfo,
+func (estimator *BinpackingNodeEstimator) Estimate(
+	pods []*apiv1.Pod,
+	nodeTemplate *schedulernodeinfo.NodeInfo,
 	upcomingNodes []*schedulernodeinfo.NodeInfo) int {
```

Comment:
This is unrelated to this PR, but is there any reason we're still passing upcomingNodes to Estimate() after changes to binpacking filterOutSchedulable? cc: @vivekbagade

Reply:
Good point. Those are not used. Removing.